src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Data/CollectionViewGroupInternal.cs

```diff
@@ -523,19 +524,20 @@ protected virtual void OnGroupByChanged()
         internal void AddSubgroupToMap(object nameKey, CollectionViewGroupInternal subgroup)
         {
             Debug.Assert(subgroup != null);
-            if (nameKey == null)
-            {
-                // use null name place holder.
-                nameKey = _nullGroupNameKey;
-            }
+
+            // Use null name place holder.
+            nameKey ??= s_nullGroupNameKey;
+
             if (_nameToGroupMap == null)
             {
-                _nameToGroupMap = new Hashtable();
```

Comment:
Since you changed nameKey to `??=`, why not here too?

Reply:
Now that's a good question :D Will adjust to `??=` too.